src/group/group.interface.ts

```diff
@@ -3,6 +3,7 @@ import User from './user/user.interface';
 export interface IGroupPrimal {
   name: string;
   description: string;
+  icon: string;
```

Comment:
Because It is not required:
```suggestion
  icon?: string;
```